clients/python/test/e2e/ci/e2e/e2e/_data_classes.py

```diff
@@ -0,0 +1,120 @@
+import configparser
+from pathlib import Path
+from typing import Dict
+
+import osparc
+from packaging.version import InvalidVersion, Version
+from pydantic import BaseModel, field_validator
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+# Holds classes for passing data around between scripts.
+
+
+class ServerConfig(BaseSettings):
+    """Holds data about server configuration"""
+
+    host: str
+    key: str
+    secret: str
+
+    model_config = SettingsConfigDict(env_prefix="osparc_api_")
+
+    def __hash__(self):
+        return hash(self.host + self.key + self.secret)
+
+    def __eq__(self, other):
+        return (
+            self.host == other.host
+            and self.key == other.key
+            and self.secret == other.secret
+        )
+
+
+def is_empty(v):
+    return v is None or v == ""
+
+
+class ClientConfig(BaseModel):
```

Comment:
Why the `ServerConfig` inherits from `BaseSettings` and not the `ClientConfig`. They are both settings, right?

TIP: I would name both using suffix `Settings`, specially if you inherit from `BaseSettings`. Note that pydnatic has model's config as well. That is one of the reasons why in osparc-simcore we avoid using "config"